migration/transformation_schema_2_to_1.yaml

```diff
@@ -0,0 +1,134 @@
+# This file provides specific mappings for metric data between Java 2.x to Java 1.x
+# The format should be the same as the otel file_format for schema files.
+file_format: 1.1.0
+# TODO: Fix URL
+schema_url: https://github.com/signalfx/splunk-otel-java/migration/transformation_schema_2_to_1.yaml
+versions:
+  # Java version 2.0.0 is the new version. The following mappings translate
+  # FROM the new/correct/stable names back to the previous names.
+  # The names to use when applying attribute renaming are the NEW names.
+  # <new_name>: <old_name>
+  # We want to translate TO the old names as a migration step
+  2.0.0:
+    metrics:
+      changes:
+        #--------------------------------------------------------------
+        # Memory metrics
+        - rename_metrics:
+            jvm.memory.used:               runtime.jvm.memory.used           # micrometer
+            jvm.memory.committed:          runtime.jvm.memory.committed      # micrometer
+            jvm.memory.limit:              runtime.jvm.memory.max            # micrometer
+            jvm.memory.used_after_last_gc: runtime.jvm.memory.usage.after.gc # micrometer
+#            jvm.memory.init:               process.runtime.jvm.memory.init  # experimental
```

Comment:
Why is this commented out?

Reply:
This, along with the other commented out lines, are the otel (non-micrometer) breaking semconv changes. I had stubbed those out first in case we also wanted to help migrate those with transforms.  I don't think this was in the micrometer metrics.